draft-ietf-quic-transport.md

```diff
@@ -7334,6 +7334,36 @@ change controller of the IETF and a contact of the QUIC working group
 (quic@ietf.org).
 
 
+## QUIC Versions Registry {#iana-version}
+
+IANA \[SHALL add/has added] a registry for "QUIC Versions" under a "QUIC"
+heading.
+
+The "QUIC Versions" registry governs a 32-bit space; see {{versions}}. This
+registry follows the registration policy from {{iana-policy}}. Permanent
+registrations in this registry are assigned using the Specification Required
```

Comment:
I don't think we should use Specification Required here. Google uses some of these codepoints in production, and we don't have `a permanent and readily available public specification, in sufficient detail so that interoperability between independent implementations is possible`. Expert Review would be a better fit.

Reply:
A provisional registration should do for the versions you have.  Any reason why it wouldn't?

Reply:
Given that these code points were deployed for huge amounts of traffic, and there will still be old devices speaking them out there for years, provisional sounds odd to me - no one will be able to reclaim these safely. But if in this context we use "provisional" to mean "doesn't have a public spec" (as opposed to the original meaning of the word) then it works?

Reply:
I think of provisional as something that's not permanent. In this case, while this will take a while to go away, it is getting "drained" off the Internet. In that sense, I would call this provisional.

Reply:
@janaiyengar that's also true of draft-29, and will be true of QUICv1 once we ship QUICv2. Why does the presence of a specification impact provisional vs permanent?

Reply:
@DavidSchinazi : It's not the presence of a spec, but that a provisional registration allows for the _absence_ of a spec. See https://quicwg.org/base-drafts/draft-ietf-quic-transport.html#section-22.